src/components/__tests__/Alert.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { createElement } from "react";
```

Comment:
This entire test case is old. Please get the latest changes from the pagination widget